lib/discovery_api_web/views/data_view.ex

```diff
@@ -86,4 +84,12 @@ defmodule DiscoveryApiWeb.DataView do
     |> Map.get("feature")
     |> Jason.decode!()
   end
+
+  defp do_transform(el, false) do
```

Comment:
Could we rename el to element or column or something? (Same for both do_transforms)
```suggestion
  defp do_transform(element, false) do
```

Reply:
sure

Reply:
👍